app/models/local_authority.rb

```diff
@@ -5,4 +5,10 @@ class LocalAuthority < ApplicationRecord
   belongs_to :api_client, optional: true
 
   validates :name, presence: true
+
+  enum name: {
```

Comment:
But if you do it this way it means you have to make a code change every time you have another council added.
Is it possible to seed the model with council names and then test against that? Then it's just a matter of changing some data / csv for each new council.

Reply:
yes that's correct! at the moment is good to have it there so we have the control. if we're going to support new local_authority we should add it here

Reply:
So imagining that we have a product used by 100s of councils and we want to switch from using enum to adding row items into the local_authorities table....

In this case ...you can switch from enum to seeding without any interface change to local_authority ??

Reply:
we are not in the stage at the moment and we want to restrict it!